README.md

```diff
@@ -29,3 +29,16 @@ This tasks will use various implementation techniques such as particles, complex
 
 ## Sprints reviews
 
+
+### Emoji for commit
```

Comment:
Don't add this to the main readme, maybe create a new readme in doc or just keep the link on the notion